libhb/handbrake/common.h

```diff
@@ -536,7 +536,7 @@ struct hb_job_s
          cfr:               0 (vfr), 1 (cfr), 2 (pfr) [see render.c]
          pass:              0, 1 or 2 (or -1 for scan)
          areBframes:        boolean to note if b-frames are used */
-#define HB_VCODEC_MASK         0x0FFFFFF
+#define HB_VCODEC_MASK         0x3FFFFFF
```

Comment:
@mstorsjo Not sure why this change?

Reply:
Because this adds the new flags `HB_VCODEC_FFMPEG_MF_H264` and `HB_VCODEC_FFMPEG_MF_H265`, so this expands the mask to include those bits in it.

Reply:
Ah, clearly I should have recognized that, given it's labeled **_MASK**. 😸